src/main/java/mezz/jei/plugins/vanilla/anvil/AnvilRecipeCategory.java

```diff
@@ -0,0 +1,50 @@
+package mezz.jei.plugins.vanilla.anvil;
+
+import mezz.jei.api.IGuiHelper;
+import mezz.jei.api.gui.IDrawable;
+import mezz.jei.api.gui.IGuiItemStackGroup;
+import mezz.jei.api.gui.IRecipeLayout;
+import mezz.jei.api.ingredients.IIngredients;
+import mezz.jei.api.recipe.BlankRecipeCategory;
+import mezz.jei.api.recipe.VanillaRecipeCategoryUid;
+import net.minecraft.init.Blocks;
+import net.minecraft.util.ResourceLocation;
+
+import javax.annotation.Nonnull;
+
+public class AnvilRecipeCategory extends BlankRecipeCategory<AnvilRecipeWrapper>
+{
+    @Nonnull
```

Comment:
Just use `@MethodsNonnullByDefault` and `@ParametersAreNonnullByDefault` in package info file.